server/server/initializers/constants.ts

```diff
@@ -1401,6 +1401,10 @@ async function buildLanguages () {
   languages['zh-Hans'] = 'Simplified Chinese'
   languages['zh-Hant'] = 'Traditional Chinese'
 
+  // Catalan languages
+  languages['ca'] = 'Catalan'
```

Comment:
I don't think you need this since the `languages` object already has this value :)